README.md

```diff
@@ -15,7 +15,7 @@ The Outline Shadowsocks service allows for:
 
```

Comment:
Maybe add to the README that little image that shows the build status like we have for outline-client and outline-server?

Reply:
Done